grammar.js

```diff
@@ -1482,6 +1483,15 @@ module.exports = grammar({
     dictionnary: ($) =>
       seq('{', commaSep($.dictionnary_entry), optional(','), '}'),
 
+    // :h literal-Dict
+    literal_dictionary_entry: ($) =>
+      seq(field('key', $.literal_key), ':', field('value', $._expression)),
```

Comment:
Can you make that an unnamed node ?
```suggestion
    _literal_dictionary_entry: ($) =>
      seq(field('key', $.literal_key), ':', field('value', $._expression)),
```

Reply:
I think I can, but wouldn't that be inconsistent with the fact that `dictionnary_entry` nodes are named?

Reply:
Done.